playbooks/openstack-preflight_osh.yml

```diff
@@ -0,0 +1,28 @@
+#
+# (c) Copyright 2019 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# Playbook to verify that required resources are in the OpenStack
+# Cloud
+
+---
```

Comment:
This doesn't seem to be a valid ansible play.  It's missing the toplevel attributes. (See also the test failure https://ci.suse.de/blue/organizations/jenkins/cloud-socok8s%2Fsocok8s/detail/PR-270/5/pipeline/34/#step-43-log-132)

Same is true for `openstack-preflight_ses.yml`

Reply:
fixed both of those.